content/en/docs/instrumentation/js/exporters.md

```diff
@@ -14,21 +14,181 @@ can learn how to setup exporters following the
 
 {{% /alert %}}
 
-## OTLP endpoint
+## Example backend
 
-To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
-Jaeger) you'll want to use an exporter package, such as
-`@opentelemetry/exporter-trace-otlp-proto`:
+{{% alert title="Note" color="info" %}}
+
+If you have a backend already set up, you can skip this section and read the
+instructions for the exporter you want to use.
+
+{{% /alert %}}
+
+If you don't have a backend setup already, you will find instructions below to
+set up an [OpenTelemetry Collector](/docs/collector/) with a basic configuration
+to send your telemetry to the console.
+
+Additionally, you will find instructions to run
+[Jaeger](https://jaegertracing.io) to visualize your traces, or
+[Prometheus](https://prometheus.io) to visualize your metrics.
+
+In this section you will find instructions to set up the
+[OpenTelemetry Collector](/docs/collector/), [Jaeger](https://jaegertracing.io)
+and [Prometheus](https://prometheus.io) to get you started quickly.
+
+### Collector (Traces, Metrics, Logs)
```

Comment:
Hmmm. This section still doesn't sit right with me. It feels like the spirit of this section is to figure out how to export OTLP from the JS SDK, but here we're configuring a collector to scrape a prometheus endpoint and accept Zipkin data, when neither of those things are documented for the JS SDK. I think I'd prefer to have this be a minimal collector that just lets you verify your OTLP data is getting exported correctly.

Reply:
> Hmmm. This section still doesn't sit right with me. It feels like the spirit of this section is to figure out how to export OTLP from the JS SDK, [...]

That's not the case anymore, I moved that section up in the hierarchy to be a "top level" section giving people the opportunity to setup a sample backend that works with all (most?) exporters coming later on this page:

<img width="271" alt="Screenshot 2023-10-20 at 12 43 43" src="https://github.com/open-telemetry/opentelemetry.io/assets/1519757/c91ec9eb-bed9-43e5-9b5c-3552c737fedd">

Ultimately I would move this out into a snippet again and make it available for all the languages.

My goal is making it easy for people to go through that documentation and if they don't have a backend yet or sending it to their production backend is complicated, they can quickly get started here

If this is too much for that page, the alternative is to move this out to some other page that can be referenced from here (not sure yet where to put it, but that's something we can figure out)